docs/design/lacp/README.md

```diff
@@ -0,0 +1,77 @@
+# Link Aggregation Control Protocol
+
+## Description
+
+Link Aggregation allows parallel full duplex point-to-point links to be used as if they were a single link. This design is focused on the following scenario.
+
+![LACP usage scenario](lacp-usage-scenario.png "LACP usage scenario")  
+
+LACP aggregator is connected to network controlled by open-kilda using one or multiple links (using one or multiple switches). From open-kilda perspective all ports connected to the same LACP aggregator should be treated as aggregateable links. Kilda should be able to:
+ * establish LACP session with aggregators
+ * create flows terminated on aggregated ports
+ 
+ Traffic from any source endpoint of such flows should be balanced over all destination endpoints which has proper LACP session established.
+
+## LACP session control
+
+![LACPDU capture](lacpdu-capture.png "LACPDU capture")
+
+Additional default openflow rule is installed on each switch to catch LACP packets and send them to the controller. This rule has match by eth_type=0x8809 (Slow protocols).
```

Comment:
rule should be mettered, worth mentioning

Reply:
fixed